Task07 – ConsumeRealData/json-reader.mjs

```diff
@@ -0,0 +1,29 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+
+function sortArray(objects){
+    objects.sort(function(obj1, obj2){
+        let x = obj1.name.toLowerCase();
+        let y = obj2.name.toLowerCase();
+        if (x < y) {return -1;}
+        if (x > y) {return 1;}
+        return 0;
+    });
+}
+
+export async function getOnePieceData() {
+    const url = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+    try {
+        let onePieceData = await fetch(url).then(response => response.json());
```

Comment:
In this case if you use async/await function you don't need to use the .then() and .catch() promise methods. So you have a return in line 25, Why do you use this method?

Reply:
Hi, I am using the .then() for the fetch promise and then I use the await async to return the result of the fetch method, and I return the data because I need it on the home-page.js file, in line 15, the mistake there I think is the await on the return is not necessary

Reply:
Ok ok thanks! I'll be changing and testing it asap :)